scripts/build-joplin-cloud.sh

```diff
@@ -9,4 +9,13 @@ if [ "$HEAD" == '2906-toc-horizontal-rules' ]; then
   export CMS_API='https://joplin-staging.herokuapp.com/api/graphql'
 fi
 
+# SAVE 👇 Example: "Let's your branchname is 2662-service-step-styling": Note trucation of name in CMS_API link.
+# if [ "$HEAD" == '2662-service-step-styling' ]; then
+#   export CMS_API='https://joplin-pr-2662-service-step-st.herokuapp.com/api/graphql'
+# fi
```

Comment:
It looks like we cleaned up some stale apps... I had to dig up and old one to copy the condition. 
- If y'all don't mind, it might be nice to have a saved exampled noted out here for future Janis/Joplin PR!?
- OR, I can wipe it!